Executables/Game/AffectManager.cs

```diff
@@ -0,0 +1,131 @@
+﻿using Dapper.Contrib.Extensions;
+using Microsoft.Extensions.Logging;
+using QuantumCore.API.Game.World;
+using Dapper;
+using System.Data;
+using Affect = QuantumCore.Database.Affect;
+using AffectAPI = QuantumCore.API.Core.Models.Affect;
+using QuantumCore.Game.Packets.Affects;
+
+namespace QuantumCore.Game;
+
+public class AffectManager : IAffectManager
+{
+    private readonly ILogger<IAffectManager> _logger;
+    private readonly IDbConnection _db;
+
+    public AffectManager(ILogger<IAffectManager> logger, IDbConnection db) {
+        _logger = logger;
+        _db = db;
+    }
+
+    public void SendAffectAddPacket(IPlayerEntity playerEntity, Affect affect, int duration)
+    {
+        var affectAdd = new AffectAdd();
+        var affectAddPacket = new AffectAddPacket
+        {
+            Type = (uint) affect.Type,
+            ApplyOn = affect.ApplyOn,
+            ApplyValue = (uint) affect.ApplyValue,
+            Duration = (uint) duration,
+            Flag = (uint) affect.Flag,
+            SpCost = (uint) affect.SpCost
+        };
+        affectAdd.Elem[0] = affectAddPacket;
+        playerEntity.Connection.Send(affectAdd);
+    }
+
+    public async Task SendAffectRemovePacket(IPlayerEntity playerEntity, long type, byte applyOn)
```

Comment:
Please rename the methods to `-Async` if they represent a `Task` or `Task<T>`. This is a convention by .NET and helps reading the code